web/modules/custom/do_generated_content/src/Generator/NodeGeneratorBase.php

```
@@ -0,0 +1,331 @@
+
+      if (!$component instanceof Paragraph) {
+        $this->helper::log('Skipped %s component: no entity to reference yet.', $bundle);
+
+        continue;
+      }
+
+      $components[] = $component;
+    }
+
+    return $components;
+  }
```

PHPMD (2.15.0)

[error] The parameter $field_name is not named in camelCase. (undefined)

(CamelCaseParameterName)